cosmosdb/management_endpoint.bal

```diff
@@ -138,14 +147,14 @@ public client class ManagementClient {
         return mapHeadersToResultType(response);
     }
 
-    # Create a container inside the given database.
+    # Creates a container inside the given database.
     #
     # + databaseId - ID of the database to which the container belongs to
     # + containerId - ID of the new container. Must be a unique value.
-    # + partitionKey - A record of type `PartitionKey`
-    # + indexingPolicy - Optional. A record of type `IndexingPolicy`.
-    # + throughputOption - Optional. Throughput parameter of type int or json.
-    # + return - If successful, returns `Container`. Else returns `Error`.
+    # + partitionKey - A record `cosmos_db:PartitionKey`
+    # + indexingPolicy - A record `cosmos_db:IndexingPolicy`.
+    # + throughputOption - Throughput parameter of type int or json.
```

Comment:
Suggestion: "A throughput parameter of `int` or `json` type"

Replace the article 'A" at the start of the sentence with "The" if relevant.